evidence/store.go

```diff
@@ -79,11 +79,12 @@ func NewEvidenceStore(db dbm.DB) *EvidenceStore {
 func (store *EvidenceStore) PriorityEvidence() (evidence []types.Evidence) {
 	// reverse the order so highest priority is first
 	l := store.listEvidence(baseKeyOutqueue, -1)
-	l2 := make([]types.Evidence, len(l))
-	for i := range l {
-		l2[i] = l[len(l)-1-i]
+	for i := len(l)/2 - 1; i >= 0; i-- {
```

Comment:
```go
i, j := 0, len(l)-1; i < j; i, j = i+1, j-1 {
  l[i], l[j] = l[j], l[i]
}
```

How about ^^^?

Reply:
I think both  is ok，respect your prefrence.